x/lockup/keeper/migration.go

```diff
@@ -65,6 +67,13 @@ func MergeLockupsForSimilarDurations(
 				continue
 			}
 			coin := lock.Coins[0]
+
+			// check if the reward receiver is the lock owner.
+			// if not, we do not normalize the lock.
+			if lock.RewardReceiverAddress != "" {
+				continue
+			}
```

Comment:
ref: https://github.com/osmosis-labs/osmosis/pull/5281/files#r1208613288

I don't understand why we use an empty string placeholder when we could use the owner address directly. Please let me know